src/ontology/mondo-edit.obo

```diff
@@ -298363,6 +298363,7 @@ is_a: MONDO:0043765 {source="DC-OMIM:612976", source="https://orcid.org/0000-000
 [Term]
 id: MONDO:0013069
 name: autosomal recessive optic atrophy, OPA7 type
+def: "Optic atrophy with or without extraocular features that is caused by a mutation in the TMEM126A gene." [https://orcid.org/0009-0007-1636-9645, OMIM:612989]
```

Comment:
Following the "genus differentia" rule for definition, we should start with the parent term is "optic atrophy".